spec/controllers/api/v1/keywords_controller_spec.rb

```diff
@@ -3,10 +3,79 @@
 require 'rails_helper'
 
 RSpec.describe API::V1::KeywordsController, type: :controller do
-  describe 'POST#create', api_authentication: :user do
+  describe 'GET#index' do
+    context 'given a request without params' do
```

Comment:
Should we add a test to expect that it return a valid keywords?

Reply:
Fixed in [`ddb5fab` (#55)](https://github.com/suho/google-search-ruby/pull/55/commits/ddb5fabf379d63d5c7e6ad4edc49abbde8fe2297)